assembly/src/todo/instruction/field_ops.rs

```diff
@@ -122,3 +122,31 @@ pub(super) fn neq_imm(
         span.add_ops([Push(*imm), Eq, Not])
     }
 }
+
+// HELPER FUNCTIONS
```

Comment:
I'm not sure why this is on helper, but it is a minor nit that can be fixed in a subsequent PR